### DISCUSSION TITLE
not creating default opt/postal/config folder "missing doc in new documents"

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#1524] [Help with using Postal] [answered]
Hello @adamcooke 

We are installing postal according to new doc and found that 

https://docs.postalserver.io/install/installation#configuration
-------------------------------------
The repository you cloned includes a tool to automatically generate some initial configuration files. This will generate three files in /opt/postal/config
------------------------------------
it is not generating "config" folder inside the /opt/postal

Please do cross check 

![image](https://user-images.githubusercontent.com/32220165/127832457-49006ba1-b74b-47e5-98bb-b6478bcd7dfd.png)

**Comment by willpower232** [comment, marked as answer]
have you actually run `postal bootstrap yourdomain.com` or whatever?

**Reply by adamcooke** [reply]
You'll need to actually follow all the instructions 😉

**Reply by willpower232** [reply]
I've made a PR to make this more obvious :nerd_face:
